src/black/trans.py

```diff
@@ -453,7 +453,7 @@ def make_naked(string: str, string_prefix: str) -> str:
             # with 'f'...
             if "f" in prefix and "f" not in next_prefix:
                 # Then we must escape any braces contained in this substring.
-                SS = re.subf(r"(\{|\})", "{1}{1}", SS)
+                SS = re.sub(r"(\{|\})", r"\1\1", SS)
```

Comment:
This will never have unicode to match right so we should be safe ... :D

Reply:
As far as I can tell the missing unicode support is just about character classes, so some "word" characters don't match `\w` in re but do in regex. That doesn't apply here.